nav2_planner/src/planner_server.cpp

```diff
@@ -450,21 +418,20 @@ PlannerServer::computePlan()
     getPreemptedGoalIfRequested(action_server_pose_, goal);
 
     // Use start pose if provided otherwise use current robot pose
-    geometry_msgs::msg::PoseStamped start;
-    if (!getStartPose(action_server_pose_, goal, start)) {
-      return;
+    if (!getStartPose<ActionToPose>(goal, start)) {
```

Comment:
Is the explicit template needed?

Reply:
Yes, 

once you remove the action server parameter it is unable to infer the template arg.